src/include/distributed/intermediate_result_pruning.h

```diff
@@ -13,6 +13,8 @@
 
 #include "distributed/subplan_execution.h"
 
+#define LOCAL_NODE_ID 555
```

Comment:
TODO: why 555? It used to be -1, but nodeId is uint, so find a better const like uint32MAX or such

Reply:
https://github.com/citusdata/citus/blob/master/src/include/distributed/master_metadata_utility.h#L84

seems to me the `nodeId` is not a uint in this struct

Reply:
fixed in another PR